pkg/sloop/webserver/debug.go

```diff
@@ -162,6 +164,107 @@ func listKeysHandler(tables typed.Tables) http.HandlerFunc {
 	}
 }
 
+type SloopKeyInfo struct {
+	TotalKeys     int64
+	EstimatedSize int64
+}
+
+type SloopKey struct {
+	TableName   string
+	PartitionID string
+}
+
+type histogram struct {
+	HistogramMap map[SloopKey]*SloopKeyInfo
+	TotalKeys    int
+	DeletedKeys  int
+}
+
+// returns IsDeletedExpired, TableName, PartitionId, EstimatedSize, error.
+func ParseSloopKey(item badgerwrap.Item) (bool, string, string, int64, error) {
```

Comment:
why pass the whole item instead of just the key?

Reply:
Because we have to check its other parameters too.